cmd/snap/cmd_repair_repairs.go

```diff
@@ -69,6 +69,10 @@ func init() {
 }
 
 func (x *cmdShowRepair) Execute(args []string) error {
+	if len(x.Positional.Repair) == 0 {
+		return fmt.Errorf("no `<repair-id>` given, did you mean 'snap repairs' to list all repairs?")
```

Comment:
why the \`\`  ?   also they can be many so perhaps `no <repair-id>s given`  ?

Reply:
fwiw, I'm not too keen on questions being asked in command output - it's like we're admitting there's an ambiguity in the design. Maybe:

`error: no <repair-id> given. Try 'snap repairs' to list all repairs or specify a specific repair id.`

Reply:
I removed the '`' now and added the 's'

Reply:
I like your suggestion a lot @degville !